dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/LogTraceData.java

```diff
@@ -10,6 +10,7 @@
  *******************************************************************************/
 package com.ibm.ws.logging.data;
 
+import java.util.Map;
```

Comment:
is this used?

Reply:
Yes, its used to create the "newAliases" maps